src/react/components/common/apiVersionPicker/apiVersionPicker.tsx

```diff
@@ -31,20 +42,22 @@ export class APIVersionPicker extends React.Component<IAPIVersionPickerProps> {
         return (
             <select id={this.props.id}
                 disabled={!constants.enableAPIVersionSelection}
-                defaultValue={constants.enableAPIVersionSelection && !this.props.value ? APIVersionPatches.patch2 : undefined}
+                defaultValue={APIVersions.versions[APIVersions.defaultIndex]}
```

Comment:
nit: Can we extract `const { versions, defaultIndex } = APIVersions;` first to reduce redundant APIVersions.xxx?